### DISCUSSION TITLE
Donation

### DISCUSSION THREAD

**Discussion by sanchosk** [OpenHantek/OpenHantek6022#212] [General]
Hi. I'd like to ask - is there some form of donation possible?
I've bought 6022BL and without this SW, I would not be able to utilize it under my normal OS.
This SW was the sole purpose I've bought the scope.
Is there a "thank you" possibility for donation? E.g. ko-fi or similar?

**Comment by Ho-Ro** [comment]
If you want to donate something, please give it to the [Free Software Foundation](https://www.fsf.org/). Without the FSF, we would not have this free open software we know today.

Thank you, Martin

**Comment by sanchosk** [comment]
Thanks, will do.
